tests/framework/icd/test_icd.cpp

```diff
@@ -70,12 +70,18 @@ bool CheckLayer(std::vector<LayerDefinition>& layers, std::string layerName) {
     return false;
 }
 
-bool IsInstanceExtensionEnabled(const char* extension_name) {
+bool IsInstanceExtensionSupported(const char* extension_name) {
     return icd.instance_extensions.end() !=
            std::find_if(icd.instance_extensions.begin(), icd.instance_extensions.end(),
                         [extension_name](Extension const& ext) { return ext.extensionName == extension_name; });
```

Comment:
Just realized that this probably will break if the compiler doesn't de-duplicate const char*'s in static storage. `string_eq()` should work buuut may not be in the headers available to test_icd.cpp.
This is definitely a nitpick and should not hold up the PR.